src/std/array.ab

```diff
@@ -1,7 +1,7 @@
 /// Returns index of the first value found in the specified array.
 ///
 /// If the value is not found, the function returns -1.
-pub fun array_first_index(array, value): Num {
+pub fun array_index(array, value): Num {
```

Comment:
I renamed this standard library function to more closely align with `array_search`. I think it's ok to do this while we're still in alpha, but further down the line we may want to have the compiler automatically use deprecated function names (with a warning) via a new annotation:
```
#[replaces_deprecated(array_first_index)]
pub fun array_index(array, value): Num {
```